extras/dynflow-sidekiq.rb

```diff
@@ -0,0 +1,25 @@
+rails_root = Dir.pwd
+
+app_file = File.expand_path('./config/application', rails_root)
+require app_file
```

Comment:
If `rails_root` is `Dir.pwd`, wouldn't this be equivalent:
```suggestion
require_relative 'config/application'
```

Reply:
Isn't `require_relative` relative to file location, not to the working directory? And I like the idea we can put this script wherever we want now.

Reply:
You're right.